.circleci/config.base.yml

```diff
@@ -148,7 +157,7 @@ jobs:
       NODE_OPTIONS: --max-old-space-size=4096
 
   lint:
-    <<: *linux-e2e-executor
+    <<: *linux-e2e-executor-large
```

Comment:
does lint need large?
I think that we could try medium for:
- lint
- publish_to_local_registry
- upload_pkg_binaries
- verify_pkg_binaries

They're not very CPU intensive I think.

Reply:
we are only billed by the minute used; those jobs all take less than 5 minutes combined (costs penies) & run in series, so I would rather keep those where they are if the 4 cores help reduce linear time